src/ngraph/node.cpp

```diff
@@ -35,19 +35,69 @@ atomic<size_t> Node::m_next_instance_id(0);
 
 Node::Node(const std::string& node_type, const NodeVector& arguments, size_t output_size)
     : m_node_type(node_type)
-    , m_instance_id(m_next_instance_id.fetch_add(1))
```

Comment:
Why move these initializers?

Reply:
So it would get picked up by all the constructors (it's in the member initializer).